Source/LoadByOS/WindowsConfigApp/FolderIcon.cs

```diff
@@ -0,0 +1,106 @@
+﻿using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.Formats.Png;
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+
+namespace WindowsConfigApp
+{
+	internal static partial class FolderIcon
+	{
+		// https://stackoverflow.com/a/21389253
+		public static byte[] ToIcon(this Image img)
```

Comment:
I remember writing this but I can't see in this PR where it was moved from.

Reply:
This is from your Dina windows library. That library uses winforms, so I couldn't use it without retaining that dependency.

Reply:
Ah, that makes sense